apps/api/src/app/groups/groups.service.test.ts

```diff
@@ -98,6 +98,19 @@ describe("GroupsService", () => {
 
             await expect(fun).rejects.toThrow("You are not the admin")
         })
+
+        it("Should update a group if the treeDepth changes", async () => {
+            const newTreeDepth = 20
+            const { treeDepth } = await groupsService.updateGroup(
+                groupId,
+                {
+                    treeDepth: newTreeDepth
+                },
+                "admin"
+            )
+
+            expect(treeDepth).toBe(newTreeDepth)
```

Comment:
A better test would be to make sure the group was actually updated in the contract, like in [here](https://github.com/privacy-scaling-explorations/bandada/blob/876c69b03c0dd694f15116abbc3c01671ecdbe78/apps/api/src/app/groups/groups.service.test.ts#LL195C11-L198C33)

Right now we are only checkin the treeDepth value is set, which would have passed even without the new change introduced in this PR.

Reply:
Yes. I will add those lines too. Thanks! 👌